tests/plugins/test_pull_base_image.py

```diff
@@ -101,7 +115,7 @@ def test_pull_base_image_plugin(parent_registry, df_base, expected, not_expected
     workflow.builder.base_image = ImageName.parse(df_base)
 
     expected = set(expected)
-    expected.add(df_base)
```

Comment:
wouldn't it exist as well, we added tag, so now is image accessible via both tags